modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
 = JBoss EAP Configuration
 
-Brief blurb about the different ways to configure JBoss EAP....
+== Overview of Standalone Server
```

Comment:
It's Standalone mode. Call it JBoss EAP Standalone mode

Reply:
updated the title